src/module/chat.ts

```diff
@@ -1,5 +1,21 @@
 import { OseActor } from "./actor/entity";
 
+function canApplyDamage(html: JQuery) {
+  if (!html.find('.dice-total').length) return false;
+  switch (game.settings.get(game.system.id, "applyDamageOption")) {
+    case CONFIG.OSE.apply_damage_options.originalTarget:
+      return !!html.find(".chat-target").last().data("id");
+    case CONFIG.OSE.apply_damage_options.targeted:
+      return !!game.user?.targets?.size;
+    case CONFIG.OSE.apply_damage_options.selected:
+      return !!canvas.tokens?.controlled.length;
+    default: {
+      console.log('unknown setting');
```

Comment:
I think we should make this more specific, and a UI warning. We *shouldn't* end up here, but if we do, it'd be helpful if a user could tell us how they got here!